data/xml/W19.xml

```diff
@@ -15877,8 +15877,10 @@ One of the references was wrong therefore it is corrected to cite the appropriat
       <author><first>Francis</first><last>Tyers</last></author>
       <author><first>Jonathan</first><last>Washington</last></author>
       <pages>24–31</pages>
-      <url hash="1fe0051c">W19-6805</url>
+      <url hash="9892f661">W19-6805</url>
       <bibkey>gokirmak-etal-2019-machine</bibkey>
+      <revision id="1" href="W19-6805v1" hash="1fe0051c"/>
+      <revision id="2" href="W19-6805v2" hash="9892f661" date="2021-11-09">responded to reviewer comments</revision>
```

Comment:
Please punctuate. Also, did the authors elaborate? This is a bit too vague.

Reply:
Done. I think they submitted the wrong version the first time. The final version addressed reviewer's comments and is intended for publication.